modules/configuring-user-level-access.adoc

```diff
@@ -0,0 +1,47 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="configuring-user-level-access_{context}"]
+= Configuring user level access
+
+[role="_abstract"]
+To manage and modify the user level access you need to configure the RBAC section in Argo CD custom resource.
```

Comment:
Missing comma after introductory infinitive phrase:

`To manage and modify the user level access, you need to configure the RBAC section in Argo CD custom resource.`